### PR TITLE
Update File Encryption Software page to cardv2

### PR DESCRIPTION
**Pull request by nitrohorse**
## Description

Resolves: #none 

#### Check List 

- [x] I have read and understand [the contributing guidelines](https://github.com/privacytoolsIO/privacytools.io/blob/master/.github/CONTRIBUTING.md).

- [x] I have [listed the source code](https://github.com/privacytoolsIO/privacytools.io/blob/master/source_code.md).

- ~~[ ] This project is [free/libre software](https://www.wikipedia.org/wiki/Free_software).~~

- ~~[ ] This project has an [associated discussion](https://github.com/privacytoolsIO/privacytools.io/issues).~~

* Netlify preview for the mainly edited page: https://deploy-preview-1191--privacytools-io.netlify.com/software/encryption-tools/

* Code Repository (if applicable): N/A

### CODE REVIEW

File: _includes/sections/file-encryption.html [hunk 1]
```diff
@@ -4,39 +4,44 @@ <h1 id="encrypt" class="anchor"><a href="#encrypt"><i class="fas fa-link anchor-
...
-  footer="OS: Windows, macOS, Linux."
-  description="VeraCrypt is a source-available freeware utility used for on-the-fly encryption. It can create a virtual encrypted disk within a file or encrypt a partition
-  or the entire storage device with pre-boot authentication. VeraCrypt is a fork of the discontinued TrueCrypt project. It was initially released on June 22, 2013. According to its developers, security improvements have been implemented and issues
-  raised by the initial TrueCrypt code audit have been addressed."
-  %}
-
-  {% include card.html color="primary"
+  description="<strong>VeraCrypt</strong> is a source-available freeware utility used for on-the-fly encryption. It can create a virtual encrypted disk within a file or encrypt a partition or the entire storage device with pre-boot authentication. VeraCrypt is a fork of the discontinued TrueCrypt project. It was initially released on June 22, 2013. According to its developers, security improvements have been implemented and issues raised by the initial TrueCrypt code audit have been addressed."
+  website="https://veracrypt.fr/"
+  forum=""
```
Comment (nitrohorse) [inline _includes/sections/file-encryption.html]: TODO

File: _includes/sections/file-encryption.html [hunk 2]
```diff
@@ -4,39 +4,44 @@ <h1 id="encrypt" class="anchor"><a href="#encrypt"><i class="fas fa-link anchor-
...
-  footer="OS: Windows, macOS, Linux, Android, BSD."
-  description="GnuPG is a GPL Licensed alternative to the PGP suite of cryptographic software. GnuPG is compliant with RFC 4880, which is the current IETF standards track specification
-  of OpenPGP. Current versions of PGP (and Veridis' Filecrypt) are interoperable with GnuPG and other OpenPGP-compliant systems. GnuPG is a part of the Free Software Foundation's GNU software project, and has received major funding from the German
-  government. <a href=\"https://gpgtools.org/\">GPGTools for macOS</a>."
-  %}
-
-  {% include card.html color="warning"
+  description="<strong>GnuPG</strong> is a GPL Licensed alternative to the PGP suite of cryptographic software. GnuPG is compliant with RFC 4880, which is the current IETF standards track specification of OpenPGP. Current versions of PGP (and Veridis' Filecrypt) are interoperable with GnuPG and other OpenPGP-compliant systems. GnuPG is a part of the Free Software Foundation's GNU software project, and has received major funding from the German government."
+  website="https://gpgtools.org/"
+  forum=""
```
Comment (nitrohorse) [inline _includes/sections/file-encryption.html]: TODO

File: _includes/sections/file-encryption.html [hunk 3]
```diff
@@ -4,39 +4,44 @@ <h1 id="encrypt" class="anchor"><a href="#encrypt"><i class="fas fa-link anchor-
...
-  footer="OS: Windows, Linux, BSD."
-  description="PeaZip is a free and open-source file manager and file archiver made by Giorgio Tani. It supports its native PEA archive format (featuring compression, multi volume
-  split and flexible authenticated encryption and integrity check schemes) and other mainstream formats, with special focus on handling open formats. It supports 181 file extensions (as of version 5.5.1).</p>
-  <p><strong>macOS alternative:</strong> <a href=\"https://www.keka.io/\">Keka</a> is a free file archiver."
-  %}
-
-</div>
+  description="<strong>PeaZip</strong> is a free and open-source file manager and file archiver made by Giorgio Tani. It supports its native PEA archive format (featuring compression, multi volume split and flexible authenticated encryption and integrity check schemes) and other mainstream formats, with special focus on handling open formats. It supports 180+ archive formats."
+  website="http://www.peazip.org"
+  forum=""
```
Comment (nitrohorse) [inline _includes/sections/file-encryption.html]: TODO

File: _includes/sections/file-encryption.html [hunk 4]
```diff
@@ -45,4 +50,5 @@ <h3>Worth Mentioning</h3>
   <li><a href="https://diskcryptor.net/">DiskCryptor</a> - A full disk and partition encryption system for Windows including the ability to encrypt the partition and disk on which the OS is installed.</li>
   <li><a href="https://gitlab.com/cryptsetup/cryptsetup/">Linux Unified Key Setup (LUKS)</a> - A full disk encryption system for Linux using dm-crypt as the disk encryption backend. Included by default in Ubuntu. Available for Windows and Linux.</li>
   <li><a href="https://hat.sh/">Hat.sh</a> - A cross-platform, serverless JavaScript web application that provides secure file encryption using the AES-256-GCM algorithm in your browser. It can also be downloaded and run offline.</a></li>
+  <li><a href="https://www.keka.io/">Keka</a> - A macOS-only, open-source file archiver (expand description).</li>
```
Comment (nitrohorse) [inline _includes/sections/file-encryption.html]: Expand on this description or remove?

File: _includes/sections/file-encryption.html (hunk 1 above)
Comment (blacklight447-ptio) [reply _includes/sections/file-encryption.html]: https://forum.privacytools.io/t/discussion-veracrypt-file-encryption/1532

File: _includes/sections/file-encryption.html (hunk 2 above)
Comment (blacklight447-ptio) [reply _includes/sections/file-encryption.html]: https://forum.privacytools.io/t/discussion-gnupg-file-encryption/1533

File: _includes/sections/file-encryption.html (hunk 3 above)
Comment (blacklight447-ptio) [reply _includes/sections/file-encryption.html]: https://forum.privacytools.io/t/discussion-peazip-file-encryption/1534

File: _includes/sections/file-encryption.html (hunk 4 above)
Comment (blacklight447-ptio) [reply _includes/sections/file-encryption.html]: a macOS-only, open-source file archiver with the ability to also encrypt files. ???

File: _includes/sections/file-encryption.html [hunk 5]
```diff
@@ -4,39 +4,44 @@ <h1 id="encrypt" class="anchor"><a href="#encrypt"><i class="fas fa-link anchor-
...
-  url="https://gpgtools.org/"
-  footer="OS: Windows, macOS, Linux, Android, BSD."
-  description="GnuPG is a GPL Licensed alternative to the PGP suite of cryptographic software. GnuPG is compliant with RFC 4880, which is the current IETF standards track specification
-  of OpenPGP. Current versions of PGP (and Veridis' Filecrypt) are interoperable with GnuPG and other OpenPGP-compliant systems. GnuPG is a part of the Free Software Foundation's GNU software project, and has received major funding from the German
-  government. <a href=\"https://gpgtools.org/\">GPGTools for macOS</a>."
-  %}
-
-  {% include card.html color="warning"
+  description="<strong>GnuPG</strong> is a GPL Licensed alternative to the PGP suite of cryptographic software. GnuPG is compliant with RFC 4880, which is the current IETF standards track specification of OpenPGP. Current versions of PGP (and Veridis' Filecrypt) are interoperable with GnuPG and other OpenPGP-compliant systems. GnuPG is a part of the Free Software Foundation's GNU software project, and has received major funding from the German government."
+  website="https://gpgtools.org/"
```
Comment (Mikaela) [inline _includes/sections/file-encryption.html]:
```suggestion
  website="https://gnupg.org/"
```

**Pull request merged**